python/full_bundle.py

```diff
@@ -173,8 +185,8 @@ def create(self) -> (int, "FullBundle"):
 
         # Find and dispatch to the appropriate BaseTechBundle.
         # If nothing is found run the BaseTechBundle
-        BaseTechBundle = base_tech.get_bundle_class(self.package_name)
-        BaseTechBundle(
+        tech_bundle = base_tech.get_bundle_class(self.package_name)
+        tech_bundle(
```

Comment:
The name `tech_bundle` doesn't really mean anything.

Reply:
May I use "base_tech_bundle" instead of "BaseTechBundle" here because of PEP8?

Reply:
Sure.

Reply:
I did this because in python `base` means the same as `abstract` and it's a little bit confused me.

Reply:
I see. FWIW, in this case the meaning isn't a `Base "Tech Bundle"`, but a `"Base Tech" Bundle`. "Base Tech" being an SDK service "base technology", like Kafka, Elasticsearch or Cassandra.